torch/testing/_benchmark.py

```diff
@@ -0,0 +1,46 @@
+from typing import Callable, List, Union
+
+import torch
+
+
+def benchmark_func(
```

Comment:
Why not use torch.utils.benchmark.Timer that is already a async-execution aware variant on top of timeit.Timer ?